tests/model_hub_tests/tensorflow/test_tf_convert_model.py

```diff
@@ -115,22 +126,30 @@ def infer_fw_model(self, model_obj, inputs):
 
         return output_dict
 
+    def clean_dir(self, dir_name: str):
+        if os.path.exists(dir_name):
+            for file_name in os.listdir(dir_name):
+                file_path = os.path.join(dir_name, file_name)
+                try:
+                    if os.path.isfile(file_path) or os.path.islink(file_path):
+                        os.unlink(file_path)
+                    elif os.path.isdir(file_path):
+                        shutil.rmtree(file_path)
+                except Exception as e:
+                    pass
```

Comment:
Not sure that we need it. We have already implemented clean for TF Hub

Reply:
I only moved the existing code to a separate method, to not duplicate it in lines 143 and 146.